### ISSUE TITLE
[1.10.2 v1.13] Client Lag

### ISSUE THREAD

**Issue by radiocrack** [whatthedrunk/allthemods#117]
Issue Description:

Stuttering lag on client.  Every few seconds client will temporarily freeze.
I was using v1.6 and had no problems for a long time and recently updated to v1.13
no major changes to my base since v1.6
Continues to happen even if moving out of loaded chunks
Base is near spawn so maybe some chunks stay loaded

Version of All the Mods:
1.10.2 v1.13

temporarily added these mods but didnt fix it
1. Optifine
2. FPS Plus
3. Temporarily Disabled WTF Exploration due to issue 115

Not sure if there is anything I can do to get more information

**Comment by whatthedrunk** [comment]
Still happening in 13c?

**Comment by radiocrack** [comment]
Issue fixed

Tested on 13c with 4GB allocated.  
I popped in and out of the nether a couple times and across the map. 
Memory hovers around 2.7-3.1GB.
I am also running at 4K resolution so might impact memory usage

While switching to soartex texture pack and client crashed out of GC memory.
To be expected when switching.  Anyway here is a convenient dump if anyone is curious.
http://pastebin.com/W39x887K

Restarted client with Soartex and ran stable with 4GB allocated
Memory still hovers around 2.7-3.1GB used
no stutter. no issues.
